plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/RemoteDockerNode.java

```diff
@@ -51,6 +52,10 @@
     private final String                         nodeHost;
     private final String                         nodeIp;
 
+    private volatile boolean isBound;
```

Comment:
This is not something exposed outside and needed for a single method, so i would stick to current approach